{{ cookiecutter.repo_name }}/README.md

```diff
@@ -146,6 +174,19 @@ These are the file artifacts associated with a run. These include the logs, conf
 
 ![mlflow](docs/imgs/mlflow_detail.png?raw=true "MLFlow Detail UI")
 
+## Documentation
+
+The repo created is automatically setup with Sphinx to build HTML documentation.  If you write your pydoc in Google Style format, the parser will automatically generate the documentation.  See [Google Style Python Docstrings](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for examples of such docstrings.  
```

Comment:
You should really line break your html lines at 88.

Reply:
Can you make some linter automatically do this for Markdown files? It's annoying to do it by hand in Visual Studio. I'm sure your vim takes care of this for you, no?